task/buildah/0.1/buildah.yaml

```diff
@@ -341,6 +347,15 @@ spec:
       buildah config -a org.opencontainers.image.base.name=${base_image_name} -a org.opencontainers.image.base.digest=${base_image_digest} $container
       buildah commit $container $IMAGE
 
+      IMAGE_INDEX="$IMAGE-index"
+      if [ -n "${DISABLE_IMAGE_INDEX}" ]; then
```

Comment:
That would have been another option to achieve this functionality, but it would result in additional tasks in users pipelines. I chose the alternative of always producing Image Indexes while ensuring that the `build-image-manifest` task supports the case (I just realized that the naming is inconsistent)...